apps/docs/src/examples/extras/gradient-texture/linear/Scene.svelte

```diff
@@ -47,7 +47,7 @@
     textureWrapT
   }: SceneProps = $props()
 
-  let stops: ColorStop[] = $derived([
+  let stops = $derived<ColorStop[]>([
```

Comment:
Hey can we have a quick talk about type declarations. I asked AI about why use the `$state<type>` and it talked about giving the compiler extra info, thought ok but checked in with svelte docs, [state types page](https://svelte.dev/docs/svelte/typescript#Typing-$state), umm didn't mention `$state<type>` way at all ... is this a preference thing?

Reply:
Not just a preference, this is more typesafe since you're passing the type to the type argument rather than asserting that this type is correct. With this method you get automatic `undefined` inference if the rune arguments are empty, etc.